test/integration/CompoundERC4626.t.sol

```diff
@@ -10,7 +10,7 @@ import {MErc20} from "@protocol/MErc20.sol";
 import {MockERC20} from "@test/mock/MockERC20.sol";
 import {Addresses} from "@test/proposals/Addresses.sol";
 import {LibCompound} from "@protocol/4626/LibCompound.sol";
-import {mipb01 as mip} from "@test/proposals/mips/mip-b01/mip-b01.sol";
+import {mipb05 as mip} from "@test/proposals/mips/mip-b05/mip-b05.sol";
```

Comment:
@ElliotFriedman are we going to be required to import every proposal into this integration test moving forward? That seems undesirable unless we can automate the generation of the multiple import statement code block somehow.

Reply:
Foundry has a `getCode` cheatcode that allows you to read in the creation bytecode from a file. With a bit of experimenting and fiddling, it's possible this could work, and then `PostProposal.sol` will need to be modified to use an environment variable to find the MIP creation code.